<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commentRow</class>
 <widget class="QWidget" name="commentRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
     <item>
      <widget class="QLabel" name="lb_profileImg">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/root/icoStudent.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_username">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>_USERNAME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_verificationMark">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="te_comment">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QLabel" name="lb_dateText">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата публикации:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_publicationDate">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>_PUBLICATIONDATE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
